packages/compose/src/hooks/use-focus-on-mount/index.js

```diff
@@ -79,6 +79,25 @@ export default function useFocusOnMount( focusOnMount = 'firstElement' ) {
 			return;
 		}
 
+		if ( typeof focusOnMountRef?.current === 'function' ) {
```

Comment:
Do we still need this to be a function? Could we just remove this functionality and rely on `firstElement` until we find a case where we need it to be a function?

Reply:
This PR is the use case 😁 

See `getFirstTabbableElement` in `Modal`.